cpp/CMakeLists.txt

```
@@ -787,7 +823,7 @@ target_compile_definitions(cudf PUBLIC SPDLOG_ACTIVE_LEVEL=SPDLOG_LEVEL_${LOGGIN
 # - link libraries --------------------------------------------------------------------------------
 
 # link targets for cuDF
-target_link_libraries(cudf arrow arrow_cuda nvrtc ${CUDART_LIBRARY} cuda ${ZLIB_LIBRARIES} ${Boost_LIBRARIES})
+target_link_libraries(cudf arrow arrow_cuda nvrtc ${CUDART_LIBRARY} cuda ZLIB::ZLIB Boost::filesystem)
```

I think we can safely remove `cuda` from here now since it was removed from `RMM`
```suggestion
target_link_libraries(cudf arrow arrow_cuda nvrtc ${CUDART_LIBRARY} ZLIB::ZLIB Boost::filesystem)
```